fw/sock_clnt.c

```diff
@@ -627,21 +626,71 @@ tfw_sock_clnt_cfgend(void)
 static int
 tfw_sock_clnt_start(void)
 {
-	int r;
+	int r = 0;
 	TfwListenSock *ls;
+	TfwListenSock *ls_reconf;
+	TfwListenSock *tmp;
 
-	if (tfw_runstate_is_reconfig())
-		return 0;
+	tfw_classifier_cleanup_inport();
```

Comment:
All the ingress connections will be lost during the reconfiguration time (I suppose RSTs will be sent).